forest/jasmine/traj2stats.py

```diff
@@ -1821,6 +1821,6 @@ def gps_stats_generate_summary(
     if parameters.save_osm_log:
         with open(
             f"{logs_folder}/locations_logs_{frequency.name.lower()}.json",
-            "wa",
+            "w",
```

Comment:
Should this be `"a"` instead?

'w'	open for writing, truncating the file first
'a'	open for writing, appending to the end of file if it exists
https://docs.python.org/3/library/functions.html#open